database/postgres/dml/build.go

```diff
@@ -100,6 +103,14 @@ SELECT count(*) as count
 FROM builds
 WHERE repo_id = ?
 AND event = ?;
+`
+	// SelectRepoBuildsByEvent represents a query to select
+	// the count of builds for by repo and event type in the database.
+	SelectRepoBuildsByEvent = `
```

Comment:
I don't think we want to add this query as it was removed with:

https://github.com/go-vela/server/pull/443